### PR TITLE
feat(roll): roll Playwright to 1.2.0-next.1595094993390

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: playwright/frame.py [hunk 1]
```diff
@@ -289,21 +328,17 @@ async def selectOption(
...
-                values=convertSelectOptionValues(values),
-                timeout=timeout,
-                noWaitAfter=noWaitAfter,
-            ),
-        )
+    ) -> List[str]:
+        params = locals_to_params(locals())
+        if "values" in params:
+            values = params.pop("values")
+            params = dict(**params, **convert_select_option_values(values))
```
Comment (mxschmitt) [inline playwright/frame.py:335]: nit: `{**params, **convert_select_option_values(values)}`
But probably just a personal preference.

File: playwright/network.py [hunk 2]
```diff
@@ -173,3 +173,18 @@ def request(self) -> Request:
...
     def frame(self) -> "Frame":
         return self.request.frame
+
+
+def serialize_headers(headers: Dict[str, str]) -> List[Header]:
+    headers_list = list()
+    for name in headers:
+        header: Header = dict(name=name, value=headers[name])
+        headers_list.append(header)
+    return headers_list
```
Comment (mxschmitt) [inline playwright/network.py:183]: that would replace 5 lines: `[{"name": name, "value": value} for name, value in x.items()]`

But probably also a personal preference.

File: playwright/network.py [hunk 3]
```diff
@@ -173,3 +173,18 @@ def request(self) -> Request:
...
+
+
+def parse_headers(headers: List[Header]) -> Dict[str, str]:
```
Comment (mxschmitt) [inline playwright/network.py:186]: same for here, but feel free to keep your solution if you think its more readable.

`return {header["name"]: header["value"] for header in headers}`

File: playwright/frame.py (hunk 1 above)
Comment (pavelfeldman) [reply playwright/frame.py:335]: I did not know that {} was more Python-ic! Happy to move across the board!

File: playwright/network.py (hunk 2 above)
Comment (pavelfeldman) [reply playwright/network.py:183]: Done. That's what happens when one doesn't know Python.

File: playwright/network.py (hunk 3 above)
Comment (pavelfeldman) [reply playwright/network.py:186]: Done

**Pull request merged**